packages/build/src/plugins/manifest/load.js

```diff
@@ -12,6 +12,10 @@ const pReadFile = promisify(readFile)
 // Load "manifest.yml" using its file path
 const loadManifest = async function({ manifestPath, package, packageJson, loadedFrom, origin }) {
   try {
+    if (manifestPath === undefined) {
+      throw new Error(`This plugin is missing a "manifest.yml"`)
```

Comment:
Can we direct users/authors to a place they can see the docs for `manifest.yml`?

Reply:
:+1: 
Fixed in 9cbf780fe447ae9c951e85c9d7a639ad3e8fa343